obspy/io/seiscomp/event.py

```diff
@@ -78,12 +80,29 @@ def _read_sc3ml(filename, id_prefix='smi:org.gfz-potsdam.de/geofon/'):
     2011-03-11T05:46:24.120000Z | +38.297, +142.373
     2006-09-10T04:26:33.610000Z |  +9.614, +121.961
     """
+    sc3ml_doc = _xml_doc_from_anything(filename)
+
+    match = re.match(
+        r'{http://geofon\.gfz-potsdam\.de/ns/seiscomp3-schema/([-+]?'
+        r'[0-9]*\.?[0-9]+)}', sc3ml_doc.tag)
+
+    try:
+        version = match.group(1)
+    except AttributeError:
+        warnings.warn('The file does not appear to be a SC3ML file. Proceed '
+                      'with caution.')
+        version = SCHEMA_VERSION[-1]
+    else:
```

Comment:
Should we raise a warning in this branch? e.g. `Schema version %s is not supported, using the most recent version`.

Reply:
A warning is already raised in the [_is_sc3ml](https://github.com/obspy/obspy/blob/ee71bd14278d4153befb2aac044d89edc7242d88/obspy/io/seiscomp/core.py#L82) method. If we also put a warning here, there will be 2 warning messages.

Reply:
Right!

Reply:
Maybe the warning should be moved from the `_is` method to the `_read` method? So that people that explicitly specify the file type during reading also see it?

Reply:
Done. I have also changed the warning to an exception.